### PR TITLE
Moved `address` fields to `agent` model

### PR DESCRIPTION
**Pull request by fluturecode**
# Changes
1. Moved address fields to agent model.
2. Removed agent model from core.
3. Created schema migrations.
4. Created data migrations.

## Purpose
To moved the address fields into the agent model.

## Testing
Run alongside the client on this branch `https://github.com/Shift3/boilerplate-client-react/tree/feature/move-address`

Closes #000

### CODE REVIEW

File: {{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/core/models.py [hunk 1]
```diff
@@ -1,9 +0,0 @@
-from django.db import models
```
Comment (joshwingreene) [inline {{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/core/models.py:1]: The only issue that I found was in regards to this file. Don't forget to delete it if it doesn't include any code.

**Review by joshwingreene — CHANGES_REQUESTED** [review]
This looks good to me. Nice work!

I did find one issue. It pertains to the models.py file that is inside of the core folder.

File: {{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/core/models.py (hunk 1 above)
Comment (fluturecode) [reply {{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/core/models.py:1]: ok thanks!

**Review by joshwingreene — APPROVED** [review]

**Review by DropsOfSerenity — APPROVED** [review]
Looks good to me, nice work.

**Pull request merged**